### PR TITLE
File added javacsript assignment7

### PR DESCRIPTION
**Pull request by vijaya1811**

### CODE REVIEW

File: vijaya/JavaScript/assignment7/ques2.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset=utf-8 />
+        <title>Display The Marks Of Students</title>
+        <style>
+        h1 {
```
Comment (nileshp1122) [inline vijaya/JavaScript/assignment7/ques2.html]: proper indent space required

File: vijaya/JavaScript/assignment7/ques2.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            Enter no.of student:
+            <input type="text" id="text1"></input>
+            Enter Marks of Students:
+            <input type="text" id="text2"></input>
+            <input type="button" value="Add" onclick="add()"></input>
+            <input type="button" value="Display" onclick="display()"></input>
+            <div class="res" id="Result"></div>
+        </div>
+        <script>
+        var size,x = 1;
```
Comment (nileshp1122) [inline vijaya/JavaScript/assignment7/ques2.html]: proper indent space required

File: vijaya/JavaScript/assignment7/ques3.html [hunk 3]
```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html>
+    <head>
+    <meta charset=utf-8 />
```
Comment (nileshp1122) [inline vijaya/JavaScript/assignment7/ques3.html]: proper indent space required

File: vijaya/JavaScript/assignment7/ques3.html [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+            <input type="button" value="Add" onclick="add()"></input>
+            <input type="button" value="Display" onclick="display()"></input>
+            Before Sorting:
+            <div class="res" id="Result"></div>
+            <input type="button" value="sort" onclick="sort()"></input>
+            After Sorting:**Bubble Sort**
+            <div class="res1" id="Result1"></div>
+        </div>
+        <script>
+        var size, x = 1;
```
Comment (nileshp1122) [inline vijaya/JavaScript/assignment7/ques3.html:51]: proper indent space required

File: vijaya/JavaScript/assignment7/ques4.html [hunk 5]
```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+    <html>
+        <head>
+        <meta charset=utf-8 />
```
Comment (nileshp1122) [inline vijaya/JavaScript/assignment7/ques4.html:4]: proper indent space required

File: vijaya/JavaScript/assignment7/ques4.html [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+                <input type="button" value="Add" onclick="add()"></input>
+                <input type="button" value="Display" onclick="display()"></input>
+                Before Sorting:
+                <div class="res" id="Result"></div>
+                <input type="button" value="sort" onclick="sort()"></input>
+                After Sorting:***Selection Sort***
+                <div class="res1" id="Result1"></div>
+            </div>
+            <script>
+            var size, x = 1;
```
Comment (nileshp1122) [inline vijaya/JavaScript/assignment7/ques4.html:56]: proper indent space required

File: vijaya/JavaScript/assignment7/ques4.html [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+                }
+                for (let k = 1; k <= size; k++) {
+                    document.getElementById("Result1").innerHTML += "Marks " + y++ + " = " + array[k] + "<br>";
+                }
+            }
+            </script>
+        </body>
+    </html>
+
+
```
Comment (nileshp1122) [inline vijaya/JavaScript/assignment7/ques4.html]: add extra line at end

File: vijaya/JavaScript/assignment7/ques5.html [hunk 8]
```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html>
+    <head>
+    <meta charset=utf-8 />
```
Comment (nileshp1122) [inline vijaya/JavaScript/assignment7/ques5.html:4]: proper indent space required

File: vijaya/JavaScript/assignment7/ques5.html [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+            <input type="button" value="Add" onclick="add()"></input>
+            <input type="button" value="Display" onclick="display()"></input>
+            Before Sorting:
+            <div class="res" id="Result"></div>
+            <input type="button" value="sort" onclick="sort()"></input>
+            After Sorting:***Insertion Sort***
+            <div class="res1" id="Result1"></div>
+        </div>
+        <script>
+        var size, x = 1;
```
Comment (nileshp1122) [inline vijaya/JavaScript/assignment7/ques5.html:56]: proper indent space required

File: vijaya/JavaScript/assignment7/triangle.html [hunk 10]
```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Triangle Demo</title>
+        <style>
+        .name {
```
Comment (nileshp1122) [inline vijaya/JavaScript/assignment7/triangle.html:9]: proper indent

File: vijaya/JavaScript/assignment7/triangle.html [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+            position: relative;
+            width: 500px;
+            background-color:rgb(92, 8, 60);
+            padding: 10px 0px 10px 0px;
+            border-radius: 15px 15px 0px 0px;
+            color: white;
+            margin-top: 80px;
+            margin-left: 400px;
+        }
+        .one{
```
Comment (nileshp1122) [inline vijaya/JavaScript/assignment7/triangle.html:19]: before { space required

**Review by nileshp1122 — CHANGES_REQUESTED** [review]

**Review by nileshp1122 — APPROVED** [review]

File: vijaya/JavaScript/assignment7/ques2.html [hunk 12]
```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset=utf-8 />
+        <title>Display The Marks Of Students</title>
+            <style>
```
Comment (Pawan1005) [inline vijaya/JavaScript/assignment7/ques2.html]: remove 1 tab space

File: vijaya/JavaScript/assignment7/ques2.html [hunk 13]
```diff
@@ -0,0 +1,64 @@
...
+                padding-top: 25px;
+                gap: 10px;
+                border-radius: 10px;
+            }
+            .res{
+                background-color: rgb(253, 247, 156);
+                width: 150px;
+                padding-left: 10px;
+            }
+            </style>
```
Comment (Pawan1005) [inline vijaya/JavaScript/assignment7/ques2.html]: remove 1 tab space

File: vijaya/JavaScript/assignment7/ques2.html [hunk 14]
```diff
@@ -0,0 +1,64 @@
...
+    </head>
+        <body>
```
Comment (Pawan1005) [inline vijaya/JavaScript/assignment7/ques2.html]: make correct indentation

File: vijaya/JavaScript/assignment7/ques2.html [hunk 15]
```diff
@@ -0,0 +1,64 @@
...
+            }
+            function display() {
+                var e = "";
+                for (var y = 1; y <=size; y++) {
+                    e += "Marks " + y + " = " + array[y] + "<br/>";
+                }
+                document.getElementById("Result").innerHTML = e;
+            }
+            </script>
+        </body>
```
Comment (Pawan1005) [inline vijaya/JavaScript/assignment7/ques2.html]: make correct indentation

File: vijaya/JavaScript/assignment7/ques3.html [hunk 16]
```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset=utf-8 />
+        <title>Display The Marks Of Students</title>
+        <style>
+        h1 {
```
Comment (Pawan1005) [inline vijaya/JavaScript/assignment7/ques3.html]: make correct indentation

**Review by Pawan1005 — CHANGES_REQUESTED** [review]
make changes asap

File: vijaya/JavaScript/assignment7/ques5.html [hunk 17]
```diff
@@ -0,0 +1,89 @@
...
+            background-color: rgb(243, 233, 96);
+            width: 150px;
+            padding-left: 10px;
+        }
+        .res1 {
+            background-color: rgb(243, 233, 96);
+            width: 150px;
+            padding-left: 10px;
+        }
+    </style>
```
Comment (Pawan1005) [inline vijaya/JavaScript/assignment7/ques5.html:38]: give proper indentation

File: vijaya/JavaScript/assignment7/ques4.html [hunk 18]
```diff
@@ -0,0 +1,91 @@
...
+                        }
+                    }
+                }
+                for (let k = 1; k <= size; k++) {
+                    document.getElementById("Result1").innerHTML += "Marks " + y++ + " = " + array[k] + "<br>";
+                }
+            }
+        </script>
+    </body>
+</html>
```
Comment (Pawan1005) [inline vijaya/JavaScript/assignment7/ques4.html:90]: remove extra lines

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Review by shubhamdalal49 — APPROVED** [review]

**Review by urmilagagare — APPROVED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**